resources/file.go

```diff
@@ -110,6 +130,32 @@ func (obj *FileRes) Validate() error {
 	return obj.BaseRes.Validate()
 }
 
+func (obj *FileRes) mode() (os.FileMode, error) {
+	m, err := strconv.ParseInt(obj.Mode, 8, 32)
+	if err != nil {
+		return os.FileMode(0), fmt.Errorf("Mode should be an octal number (%v): %v", obj.Mode, err.Error())
+	}
+	return os.FileMode(m), nil
+}
+
```

Comment:
Needs short comment please about what this function does. Follow the standard format of:

// uid does blah blah blah.